cypress/e2e/article.cy.js

```diff
@@ -1,13 +1,33 @@
-describe('', () => {
-  before(() => {
+const { faker } = require('@faker-js/faker');
 
+describe('Article CRUD', () => {
+  before(() => {
   });
+  const articleTitle = faker.word.words(1);
+  const articleTopic = faker.word.words(2);
+  const articleDescription = faker.word.words(5);
+  const articleTag = faker.word.words(1);
 
-  it('', () => {
-
+  it('should create new article', () => {
+    cy.task('generateUser').then((user) => {
+      cy.login(user.email, user.username, user.password);
+    });
+    cy.contains('.nav-link', 'New Article').click();
+    cy.get('[placeholder="Article Title"]').type(articleTitle);
+    cy.get('[placeholder="What\'s this article about?"]').type(articleTopic);
+    cy.get('[placeholder="Write your article (in markdown)"]')
+      .type(articleDescription);
+    cy.get('[placeholder="Enter tags"]').type(`${articleTag}{enter}`);
+    cy.contains('.btn', 'Publish Article').click();
+    cy.url().should('contain', 'article/');
   });
 
-  it('', () => {
-
+  it('shoud delete created article', () => {
+    cy.task('generateUser').then((user) => {
+      cy.login(user.email, user.username, user.password);
+    });
+    cy.createArticle(articleTitle, articleTopic, articleDescription);
+    cy.contains('.btn', 'Delete Article').click();
+    cy.url().should('contain', 'https://conduit.mate.academy/');
   });
```

Comment:
In this test, you're deleting an article. However, the test does not verify if the article was actually deleted. After clicking the 'Delete Article' button, you should check if the article is no longer present. This could be done by trying to navigate to the article's URL and checking if the appropriate 'not found' message is displayed.